realm/realm-library/build.gradle

```diff
@@ -24,9 +24,14 @@ android {
     }
 }
 
+repositories {
+    maven { url "https://jitpack.io" }
+}
+
 dependencies {
     provided 'io.reactivex:rxjava:1.1.0'
     compile project(':realm-annotations')
+    compile 'com.github.KeepSafe:ReLinker:1.1'
```

Comment:
Can we find a way to centralise the version of ReLinker? It's used both here and in `bintrayAar.gradle` so it's easy to forget one of the two